modules/Oculus_module/src/OculusModule.cpp

```diff
@@ -956,8 +1085,8 @@ bool OculusModule::updateModule()
         // get the transformation form the oculus
         if (!getTransforms())
         {
-            yError() << "[OculusModule::updateModule] Unable to get the transform";
-            return false;
+            yError() << "[OculusModule::updateModule] Unable to get the transforms. Skipping iteration.";
```

Comment:
should we change it to a warning?

Reply:
Done in https://github.com/robotology/walking-teleoperation/pull/121/commits/ea931bad41b2795a31f7a419d2dbfc74b10909d3